Readability.js

```diff
@@ -1629,17 +1630,17 @@ Readability.prototype = {
         }
 
         var linkDensity = this._getLinkDensity(tagsList[i]);
-        var contentLength = this._getInnerText(tagsList[i]).length;
+        var contentLength = this._getInnerText(tagsList[i], true).length;
         var toRemove = false;
         if (img > p && !this._hasAncestorTag(tagsList[i], "figure")) {
           toRemove = true;
         } else if (li > p && tag !== "ul" && tag !== "ol") {
```

Comment:
It looks like this already has a similar check that could be normalized to use `isList` as well.